graylog2-web-interface/src/components/sidecars/administration/CollectorConfigurationModal.tsx

```diff
@@ -214,7 +214,11 @@ const CollectorConfigurationModal = ({
         </ModalTitle>
       </Modal.Header>
       <Modal.Body>
-        <SearchForm query={searchQuery} onQueryChange={(q) => setSearchQuery(q)} topMargin={0} queryWidth="100%" />
+        <SearchForm query={searchQuery}
```

Comment:
I think here we should use styled-component instead of introducing new props to the `SeachForm` component: 
- We can create a StyledSearchForm as below
```
const StyledSearchForm = styled(SearchForm)`
  .form-group, .query {
    width: 100% !important;
  }
`;
```
- then use it here:
```suggestion
        <StyledSearchForm query={searchQuery}
```

Reply:
Then we can remove the passed props below here and also from the `SearchForm` component.